code/modules/mining/mine_items.dm

```diff
@@ -1120,27 +1120,33 @@ var/list/total_extraction_beacons = list()
 /******************************Ore Summoner*******************************/
 
 /obj/item/weapon/oreportal
-	name = "ore summoner"
-	contained_sprite = 1
-	icon = 'icons/obj/mining_contained.dmi'
-	icon_state = "supermagneto"
-	item_state = "jaunter"
-	desc = "A handheld device that creates a well of warp energy that teleports minerals of a very specific type, size, and state to its user."
-	w_class = 3
-	force = 15
-	throwforce = 5
-	origin_tech = list(TECH_BLUESPACE = 4, TECH_ENGINEERING = 3)
+    name = "ore summoner"
```

Comment:
You started using spacebar for indentation. This is not our style. Please re-indent the file with tab.